lib/transaction.js

```diff
@@ -225,12 +232,11 @@ class Transaction extends EventEmitter {
       .then((connection) => {
         connection.__knexTxId = this.txid;
 
-        return (this._previousSibling
-          ? this._previousSibling.catch(() => {})
-          : Promise.resolve()
-        ).then(function() {
-          return connection;
-        });
+        return this._previousSibling
```

Comment:
acquireConnection also has overrides in several dialects, check them please

Reply:
The `Transaction`'s `acquireConnection` method delegates to the underlying `Dialect`'s `acquireConnection` method:

https://github.com/knex/knex/blob/72886086b65583b395d2c34851ef60a5a6d5fd75/lib/transaction.js#L216-L224

So, there isn't actually an inheritance relationship there.

Reply:
Actually, nvm.  I see what you mean. I'll check it in a little bit.

(Related note: seems like there is a lot of duplication within those implementations)

Reply:
So, the other implementations are actually ignoring the details around `this._previousSibling`.  That is potentially a problem, but not a problem that was introduced by this specific change.